components/02-components/is-active/is-active.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "label": "Is active"
```

Comment:
I suggest that we label this `Is active committee status`. We are using is-active classes for menu items, so it would be good to differentiate which one this is for.

Reply:
Or just `committee status` and also show an example of a terminated committee?